release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2468,6 +2468,47 @@ index a90ab14ff..f7f6f1a71 100644
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=2097579[*BZ#2097579*])
 
+[id="ocp-4-11-ran-known-issues"]
+== {product-title} 4.11 RAN known issues
+
+* After deploying a {sno} with the RAN DU profile, the `openvswitch: cpu_id mismatch with handler threads` Open vSwitch kernel logs are continuously generating. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-46165[*OCPBUGSM-46165*])
+
+* If secure boot is currently disabled and you attempt to enable it through ZTP, the installation will not begin the first time ZTP is started. When secure boot is enabled through ZTP, the boot options are configured before virtual CD is attached. Therefore, the first boot is booting into the existing HD with secure boot on. The installation then gets stuck because booting from CD is never triggered. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])
+
+* If an invalid subscription channel is specified in the subscription policy used by the update, the {cgu-operator-full} indicates a successful update right after the policy is enforced because the state does not change (`AtLatestKnown`). There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-43618[*OCPBUGSM-43618*])
```

Comment:
```suggestion
* If an invalid subscription channel is specified in the subscription policy that is used to perform a cluster upgrade, the {cgu-operator-full} indicates a successful update right after the policy is enforced because the state does not change (`AtLatestKnown`). There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-43618[*OCPBUGSM-43618*])
```